frontend/javascripts/messages.js

```diff
@@ -209,8 +209,8 @@ instead. Only enable this option if you understand its effect. All layers will n
   "task.no_tasks_to_download": "There are no tasks available to download.",
   "dataset.upload_success": "The dataset was uploaded successfully.",
   "dataset.upload_failed": "The dataset upload failed.",
-  "dataset.upload_none_zip_error":
-    "It looks like the selected file is not a zip file. WebKnossos only supports uploading zipped datasets. Please ensure that your dataset is compressed to a zip archive.",
+  "dataset.unsupported_file_type":
+    "It looks like the selected file is not supported. WebKnossos only supports uploading zipped WKW datasets or image files.",
```

Comment:
Please note that right now only tiff stacks are supported (until https://github.com/scalableminds/webknossos-cuber/pull/268 is finished and integrated into https://github.com/scalableminds/webknossos-worker/blob/master/tasks.py

Reply:
I left it vague on purpose, since I didn't want to introduce a dependency to the cuber (if the cuber supports a new format, this would need to be updated here). Instead, I'd rather focus on getting the `cuber` to support 99% of the use cases. But feel free to voice a different opinion :)

Reply:
Theoretically, one would also need to explain which file structures/hierarchies are supported. This opens up a can of worms in my opinion.

Reply:
Yes that makes sense, this error message might not be the best spot for this. Still, from a user perspective, I think it would be very helpful to see the supported formats somewhere. Ideally, the cuber should be documenting this and wK could link there. If such a page doesn't exist yet, we can add and link to it later.